src/Avalonia.Base/Input/PhysicalKeyExtensions.cs

```diff
@@ -0,0 +1,200 @@
+            PhysicalKey.KeyP => Key.P,
+            PhysicalKey.KeyQ => Key.Q,
+            PhysicalKey.KeyR => Key.R,
+            PhysicalKey.KeyS => Key.S,
+            PhysicalKey.KeyT => Key.T,
+            PhysicalKey.KeyU => Key.U,
+            PhysicalKey.KeyV => Key.V,
+            PhysicalKey.KeyW => Key.W,
+            PhysicalKey.KeyX => Key.X,
+            PhysicalKey.KeyY => Key.Y,
+            PhysicalKey.KeyZ => Key.Z,
+            PhysicalKey.Minus => Key.OemMinus,
+            PhysicalKey.Period => Key.OemPeriod,
+            PhysicalKey.Quote => Key.Oem7,
+            PhysicalKey.Semicolon => Key.Oem1,
+            PhysicalKey.Slash => Key.Oem2,
+
+            // Functional Keys
+            PhysicalKey.AltLeft => Key.LeftAlt,
+            PhysicalKey.AltRight => Key.RightAlt,
+            PhysicalKey.Backspace => Key.Back,
+            PhysicalKey.CapsLock => Key.CapsLock,
+            PhysicalKey.ContextMenu => Key.Apps,
+            PhysicalKey.ControlLeft => Key.LeftCtrl,
+            PhysicalKey.ControlRight => Key.RightCtrl,
+            PhysicalKey.Enter => Key.Enter,
+            PhysicalKey.MetaLeft => Key.LWin,
+            PhysicalKey.MetaRight => Key.RWin,
+            PhysicalKey.ShiftLeft => Key.LeftShift,
+            PhysicalKey.ShiftRight => Key.RightShift,
+            PhysicalKey.Space => Key.Space,
+            PhysicalKey.Tab => Key.Tab,
+            PhysicalKey.Convert => Key.ImeConvert,
+            PhysicalKey.KanaMode => Key.KanaMode,
+            PhysicalKey.Lang1 => Key.HangulMode,
+            PhysicalKey.Lang2 => Key.HanjaMode,
+            PhysicalKey.Lang3 => Key.DbeKatakana,
+            PhysicalKey.Lang4 => Key.DbeHiragana,
+            PhysicalKey.Lang5 => Key.OemAuto,
+            PhysicalKey.NonConvert => Key.ImeNonConvert,
+
+            // Control Pad Section
+            PhysicalKey.Delete => Key.Delete,
+            PhysicalKey.End => Key.End,
+            PhysicalKey.Help => Key.Help,
+            PhysicalKey.Home => Key.Home,
+            PhysicalKey.Insert => Key.Insert,
+            PhysicalKey.PageDown => Key.PageDown,
+            PhysicalKey.PageUp => Key.PageUp,
+
+            // Arrow Pad Section
+            PhysicalKey.ArrowDown => Key.Down,
+            PhysicalKey.ArrowLeft => Key.Left,
+            PhysicalKey.ArrowRight => Key.Right,
+            PhysicalKey.ArrowUp => Key.Up,
+
+            // Numpad Section
+            PhysicalKey.NumLock => Key.NumLock,
+            PhysicalKey.Numpad0 => Key.NumPad0,
+            PhysicalKey.Numpad1 => Key.NumPad1,
+            PhysicalKey.Numpad2 => Key.NumPad2,
+            PhysicalKey.Numpad3 => Key.NumPad3,
+            PhysicalKey.Numpad4 => Key.NumPad4,
+            PhysicalKey.Numpad5 => Key.NumPad5,
+            PhysicalKey.Numpad6 => Key.NumPad6,
+            PhysicalKey.Numpad7 => Key.NumPad7,
+            PhysicalKey.Numpad8 => Key.NumPad8,
+            PhysicalKey.Numpad9 => Key.NumPad9,
```

Comment:
Some of these enum values can be adjusted to be more similar to the Key enum.
Like, difference between Numpad0 and NumPad0 is unnecessary.

Reply:
I'll change them. The members are directly coming from the W3C list, with case preserved, but the inconsistency doesn't look good.

Reply:
Renamed:
 `NumpadX` ⇒ `NumPadX` (casing consistency)
 `KeyX` ⇒ `X` (avoids duplicate `Key` in qualified names); alternatively, `LetterX` could be used (consistent with `DigitX`)

Not renamed, to discuss:
 - `DigitX` ⇒ `DX`, I personally find Digit more obvious.
 - `ShiftLeft`/`ShiftRight` (also `Control`, `Alt`) ⇒ `LeftShift`/`RightShift`. Having the key name first seems more discoverable. Ideally the Left/Right `Key` members should be obsoleted in the future and be `Shift`/`Control`/`Alt` only, with the `PhysicalKey` used to distinguish between Left or Right if needed.
 - `MetaLeft`/`MetaRight` ⇒ `LWin`/`RWin`. `Meta` is platform-independent and L and R don't even match the other existing `Left`/`Right` members.